clusters/build-clusters/02_cluster/openshift-apiserver/README.md

```diff
@@ -0,0 +1,43 @@
+# openshift-apiserver
```

Comment:
Why do we have a new subfolder with only a readme? This is super confusing. Essentially this is part of the cluster setup, so why not simply put it in `release/clusters/build-clusters/02_cluster/README.md`?

Same comment for the other readme.

Reply:
<img width="251" alt="Screen Shot 2020-06-17 at 4 46 36 PM" src="https://user-images.githubusercontent.com/4013349/84948747-276bc000-b0ba-11ea-8fef-8797fb8a1c12.png">

I moved certificates to the folder as you suggested https://github.com/openshift/release/pull/9402#discussion_r441739611

Reply:
`02_cluster/README.md` would be too long.

Reply:
My suggestion was to remove all of it into a dedicated file, not to add a new folder with one file and one readme.

I strongly prefer a long readme over one I am not going to find.

Reply:
ACK. Doing it.

Reply:
Done.